schema/schema2025-08-22.yaml

```
@@ -0,0 +1,1152 @@
+      tags:
+      - Insults
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OptimizedInsult'
+              examples:
+                MyInsultsListExample:
+                  value:
+                  - reference_id: GIGGLE_Nzk1
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 20 but found 18

(indentation)